conf/register.routes

```diff
@@ -10,6 +10,7 @@ GET         /submission-drafts/:draftId/main                controllers.Submissi
 POST        /submission-drafts/:draftId/main                controllers.SubmissionDraftController.setSection(draftId: String, sectionKey: String = "main")
 
 GET         /submission-drafts                              controllers.SubmissionDraftController.getDrafts()
+POST        /submission-drafts/:draftId                     controllers.SubmissionDraftController.adjustDraft(draftId: String)
```

Comment:
Can we make it more specific to track the response in kibana for users for that draft id?